lib/etcdv3.rb

```diff
@@ -45,6 +45,18 @@ def token
     request.token
   end
 
+  def key
+    File.read(File.expand_path(@options.fetch(:key)))
+  end
+
+  def cert
+    File.read(File.expand_path(@options.fetch(:cert)))
+  end
+
+  def cacert
+    File.read(File.expand_path(@options.fetch(:cacert)))
+  end
+
```

Comment:
Hmm, wouldn't it make more sense to move all those attribute readers to the private section altogether. They're only needed by the class itself and would otherwise give access to internal state which probably shouldn't be accessed by anybody directly anyway. Or am i overlooking something here?